activemerchant.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
 
   s.has_rdoc = true if Gem::VERSION < '1.7.0'
 
-  s.add_dependency('activesupport', '>= 3.2.14', '< 5.1')
+  s.add_dependency('activesupport', '>= 3.2.14', '< 5.2')
```

Comment:
Well, what do you think is best? I was mostly just asking for opinions. 

As someone integrating ActiveMerchant, I'd personally find it frustrating that I couldn't test edge Rails versions without some dependency monkeying.

I like what you've proposed as a solution to that.